rules/falco_rules.yaml

```diff
@@ -3054,6 +3054,16 @@
   priority: WARNING
   tags: [container, cis, mitre_lateral_movement]
 
+- rule: Unprivileged Delegation of Page Faults Handling to a Userspace Process
+  desc: Detect a successful unprivileged userfaultfd syscall which might act as an attack primitive to exploit other bugs
+  condition: >
```

Comment:
just had another thought,

1. Maybe we should limit the attack scenario in container as gaining into the node is a high privileged access. 
2. If we can assume the attack scenario in container, we may want to detect `userfaultfd` by all users including root, especially people still run `root` containers quite commonly nowadays.

Reply:
@leodido let me know what you think